integrations/google.py

```diff
@@ -33,14 +38,38 @@ def GoogleHome(page, automation):
             page.click("text=Not now")
             logging.info("Clicked 'Not now' for simplify sign-in prompt")
 
-        page.click("text=OK")
-        logging.info("Clicked 'OK' after logging in")
-        page.wait_for_load_state('load')
+    if first_login:
+        try:
+            time.sleep(2)
+            # page.wait_for_selector("label:has-text(\"Don't show again\")", timeout=300)
```

Comment:
You meant to uncomment this right? if not can you remove?

Reply:
Meant to remove. good catch!

Reply:
Removed this and the one on line 52. Pushing soon